app/views/registration_mailer/ticket.html.erb

```diff
@@ -49,6 +49,9 @@
 
   <p>If you have any problems, you can contact us via mail: <%= mail_to @registration.event.contact_email, @registration.event.contact_email %></p>
 
+  <% if @event.unsubscribe_option %>
+    <p>If you cannot come: <%= link_to "uitschrijven", unsubscribe_path(id: @registration.id, barcode: @registration.barcode)  %></p>
```

Comment:
Should be in english, I would also word it as "Unsubscribe if you're unable to attend." with unsubscribe as link.